doc/api/stream.md

```diff
@@ -1761,6 +1765,8 @@ changes:
     [`stream._destroy()`][writable-_destroy] method.
   * `final` {Function} Implementation for the
     [`stream._final()`][stream-_final] method.
+  * `construct` {Function} Implementation for the
+    [`stream._construct()`][writable-_construct] method.
```

Comment:
Ditto, forget to document the `destroy`?

Reply:
`destroy` is not part of this PR?

Reply:
But it seems exists on your example :)

Reply:
Yes, it already exists :). Look a few lines above.